### PR TITLE
Default values description for builders

### LINKED ISSUE #182: Write about default values in all Builders

### PR DESCRIPTION
**Pull request by nikitin-da**
Closes #182 
@artem-zinnatullin PTAL

### CODE REVIEW

File: storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/get/PreparedGetListOfObjects.java [hunk 1]
```diff
@@ -181,7 +182,8 @@ public void call(Subscriber<? super List<T>> subscriber) {
         }
 
         /**
-         * Specifies {@link RawQuery} for Get Operation, you can use it for "joins" and same constructions which are not allowed in {@link Query}
+         * Specifies {@link RawQuery} for Get Operation,
```
Comment (artem-zinnatullin) [inline storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/get/PreparedGetListOfObjects.java]: Hm, for this case `query` or `rawQuery` is required, I think we should notice that in javadoc

Comment (artem-zinnatullin) [inline storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/get/PreparedGetListOfObjects.java]: Do you plan to fix it or I should merge PR without it?

Comment (nikitin-da) [inline storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/get/PreparedGetListOfObjects.java]: I plan to do this with #218

**Comment by nikitin-da** [comment]
I tried to create one. @artem-zinnatullin any thoughts?

**Comment by artem-zinnatullin** [comment]
Okay, merging this

**Comment by artem-zinnatullin** [comment]
Oh shit, I didn't see changes, I thought you would create separate PR for that, reviewing

**Comment by artem-zinnatullin** [comment]
Awesome, just tried to write get().cursor() and I see how it helps avoid non-passing query in compile time!

Great job! Hope to see PRs for each Builder! (please make separate PR for each Builder, it'll be easier to review them)

**Comment by nikitin-da** [comment]
Yes sir

**Comment by artem-zinnatullin** [comment]
Sir?

пн, 6 апр. 2015, 2:54, Dmitrii Nikitin <notifications@github.com>:

**Comment by nikitin-da** [comment]
it was a joke, passed=)

**Comment by artem-zinnatullin** [comment]
Lol, okay. GitHub is for serious people, no jokes!

пн, 6 апр. 2015, 2:59, Dmitrii Nikitin <notifications@github.com>:

**Pull request merged**